### ISSUE TITLE
Cannot deserialise LAW entities

### ISSUE THREAD

**Issue by benhachey** [explosion/spaCy#1427]
I can't get LAW entities to deserialise, e.g.:
```python
import spacy
from spacy.tokens import Span

text = u'Netflix is hiring a new VP of global policy according to Section 1 of the corporate charter.'

nlp = spacy.load('en_core_web_sm')
doc = nlp(text)

from spacy.tokens import Doc
from spacy.vocab import Vocab

bin_doc = doc.to_bytes()
doc2 = Doc(Vocab()).from_bytes(bin_doc)

print([(e.label_, e.text) for e in doc.ents])
print([(e.label_, e.text) for e in doc2.ents])
```

Raises:
```
[('ORG', 'VP'), ('LAW', 'Section 1')]
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-28-35776b8a45af> in <module>()
      1 print([(e.label_, e.text) for e in doc.ents])
----> 2 print([(e.label_, e.text) for e in doc2.ents])

doc.pyx in spacy.tokens.doc.Doc.ents.__get__()

span.pyx in spacy.tokens.span.Span.__cinit__()

AssertionError: 4035656307355538346
```

Breaking at:
https://github.com/explosion/spaCy/blob/485c4f6df5763a01b90117632114f96e28c31738/spacy/tokens/span.pyx#L60

**Comment by honnibal** [comment]
Bugger, thank.

To keep you working: `nlp.vocab.strings.add('LAW')`

Strings are hashed into uint64, which is obviously one-way. So if we try to look up a string that's missing it will cause an error.

**Comment by ines** [comment]
> I'm sort of worried about the implications of this though. Does this mean there were zero LAW entities in the data?

I just checked and even though it's present in the OntoNotes 5 corpus, spaCy currently doesn't have a `LAW` symbol 😱 It's also not part of our documented NER scheme... which I guess was based on `spacy.symbols`, so we never noticed? It's very mysterious.

The good news is, the issue that made it very difficult to add new symbols in v1.x is not a problem in v2.0 anymore. So we can simply add it to `spacy.symbols` now.

**Comment by ines** [comment]
Fixed and will be included in v2.0!
